From a777e0713172080b4acdcd601dba4fcb3965925d Mon Sep 17 00:00:00 2001
From: Jake Hunsaker <jhunsake@redhat.com>
Date: Wed, 7 Nov 2018 15:47:46 -0500
Subject: [PATCH] [sos_collector] Fix cluster option reporting

No longer call a removed method that handled loading cluster profiles
before that was moved into the Configuration() class.

Signed-off-by: Jake Hunsaker <jhunsake@redhat.com>
---
 soscollector/sos_collector.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/soscollector/sos_collector.py b/soscollector/sos_collector.py
index e7f3ea7..70a3dc8 100644
--- a/soscollector/sos_collector.py
+++ b/soscollector/sos_collector.py
@@ -48,13 +48,13 @@ class SosCollector():
         self.master = False
         self.retrieved = 0
         self.need_local_sudo = False
+        self.clusters = self.config['cluster_types']
         if not self.config['list_options']:
             try:
                 if not self.config['tmp_dir']:
                     self.create_tmp_dir()
                 self._setup_logging()
                 self.log_debug('Executing %s' % ' '.join(s for s in sys.argv))
-                self.clusters = self.config['cluster_types']
                 self.log_debug("Found cluster profiles: %s"
                                % self.clusters.keys())
                 self.log_debug("Found supported host types: %s"
@@ -63,8 +63,6 @@ class SosCollector():
                 self.prep()
             except KeyboardInterrupt:
                 self._exit('Exiting on user cancel', 130)
-        else:
-            self._load_clusters()
 
     def _setup_logging(self):
         # behind the scenes logging
-- 
2.14.4

